packages/core/src/config/utilityTypes.ts

```diff
@@ -0,0 +1,51 @@
+import type { Abi, AbiEvent, FormatAbiItem } from "abitype";
```

Comment:
I wonder if this file should go into the `types/` folder. Either way, we should come up with a convention for what to name and where to put type-only files.